### PR TITLE
API Request Parameters display customization

### PR DESCRIPTION
**Pull request by ppal-zededa**
API Request Parameters display customization

### Description
API Request parameters are presented in a tabular format with 5 columns
(Name, In, Required, Type and Description)
instead of 2 columns with multiple fields displayed in first column.

### Motivation and Context
We wanted to make changes to default swagger-ui layout for better usability.

### How Has This Been Tested?
Generated dist/swagger-ui* files are used in zedcloud zswagger-ui service.

### Screenshots (if appropriate):
Screen Shot 2020-08-24 at 9 51 45 AM

### My PR contains... 
- [ ] No code changes (`src/` is unmodified: changes to documentation, CI, metadata, etc.)
- [ ] Dependency changes (any modification to dependencies in `package.json`)
- [ ] Bug fixes (non-breaking change which fixes an issue)
- [X] Improvements (misc. changes to existing features)
- [ ] Features (non-breaking change which adds functionality)

### My changes...
- [ ] are breaking changes to a public API (config options, System API, major UI change, etc).
- [ ] are breaking changes to a private API (Redux, component props, utility functions, etc.).
- [ ] are breaking changes to a developer API (npm script behavior changes, new dev system dependencies, etc).
- [X] are not breaking changes.

### Documentation
- [X] My changes do not require a change to the project documentation.
- [ ] My changes require a change to the project documentation.
- [ ] If yes to above: I have updated the documentation accordingly.

### Automated tests
- [X] My changes can not or do not need to be tested.
- [ ] My changes can and should be tested by unit and/or integration tests.
- [ ] If yes to above: I have added tests to cover my changes.
- [ ] If yes to above: I have taken care to cover edge cases in my tests.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/core/json-schema-components.jsx [hunk 1]
```diff
@@ -299,15 +299,15 @@ export class JsonSchemaArrayItemFile extends Component {
   }
 
   render() {
-    let { getComponent, errors, disabled } = this.props
+    let { getComponent, errors, hidden } = this.props
```
Comment (ssat-zededa) [inline src/core/json-schema-components.jsx:302]: Why do this change from `disabled` to `hidden` especially as the use remains the same? Changing names makes it harder to sync up future changes.

**Review by ssat-zededa — CHANGES_REQUESTED** [review]

File: src/core/json-schema-components.jsx (hunk 1 above)
Comment (ppal-zededa) [reply src/core/json-schema-components.jsx:302]: We do not want to show input boxes (even in disabled mode) in "view" mode.
They should appear only when user is trying to make API call to try out the API.

Comment (udit-zededa) [reply src/core/json-schema-components.jsx]: @ssat-zededa As per your suggestion, I made changes. Please take a look.

**Review by ssat-zededa — APPROVED** [review]
